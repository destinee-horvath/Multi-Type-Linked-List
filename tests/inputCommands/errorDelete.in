DELETE -1 0
DELETE 0
DELETE element
delete 
