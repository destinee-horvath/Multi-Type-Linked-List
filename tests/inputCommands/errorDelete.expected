INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
INVALID COMMAND: INPUT
